linux_gui/constants.py

```diff
@@ -76,7 +76,7 @@
     <defaults>
       <allow_any>no</allow_any>
       <allow_inactive>no</allow_inactive>
-      <allow_active>auth_admin_keep</allow_active>
+      <allow_active>auth_admin</allow_active>
```

Comment:
So this is forcing the user to type password all the time right? Is there some intermediate way? E.g. to remember password for the current X session (unless screen locked or something similar?)